src/fontra/client/web-components/ui-form.js

```diff
@@ -207,6 +207,11 @@ export class Form extends SimpleElement {
       this._fieldChanging(fieldItem, value, undefined);
     };
     this._fieldGetters[fieldItem.key] = () => inputElement.value;
+    if (inputElement.value === "") {
```

Comment:
I think it's better to solve this in panel-selection-info.js, by adding this:
```
            disabled: glyphController.leftMargin == undefined,
```
to fieldX and fieldY (after `value:`, before `getValue:`).

Reply:
Done and tested.